esp/smc/SMCLib/TpContainer.cpp

```diff
@@ -500,7 +500,7 @@ void CTpWrapper::getServices(double version, const char* serviceType, const char
     {
         IPropertyTree& service = itr->query();
         //Only show the public services for now
-        if (!service.getPropBool("@public"))
+        if (!service.getPropBool("@public", false))
```

Comment:
super trivial: we don't normally add the default param in cases like this.

Reply:
I was trying to revert the code to the way it was (and failed). I'll take it out